storage.go

```diff
@@ -369,5 +367,16 @@ func slicesEqual(s1, s2 []byte) bool {
 	return true
 }
 
+// dataKey generates a dataKey value based on the SKYDB_ENTROPY environment
+// variable.
+func dataKey() (crypto.Hash, error) {
+	ent, err := base64.StdEncoding.DecodeString(os.Getenv("SKYDB_ENTROPY"))
```

Comment:
probably want to extract this env validation seeing as we have it twice already

Reply:
Good point. I took it a step further and moved the entire extraction of the entropy to a helper func.